djstripe/contrib/rest_framework/views.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+# Python import
+
+# Django import
+
+# Contrib import
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.views import APIView
+
+# Project import
+from ...settings import subscriber_request_callback, CANCELLATION_AT_PERIOD_END
+from ...models import Customer
+from .serializers import SubscriptionSerializer, CreateSubscriptionSerializer
+
+
+class SubscriptionRestView(APIView):
+    """
+    A REST API for Stripes implementation in the backend
+    """
+
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, format=None):
+        """
+        Return the users current subscription
+        """
+        try:
+            customer = self.request.user.customer
```

Comment:
use ``Customer.get_or_create(subscriber=self.request.user)`` here. There's a chance that the user object has no customer object attached to it.